lib/iamvery/phoenix/live_view/test_helpers.ex

```diff
@@ -16,6 +16,11 @@ defmodule Iamvery.Phoenix.LiveView.TestHelpers do
         {conn, {:ok, view, html}}
       end
 
+      def browser({conn, {:ok, view, html}}) do
+        open_browser(view)
```

Comment:
Fwiw, I've been using Kernal.tap for these sorts of things. I think it solves the problem and helps avoid adding a bunch of extra little functions to the library.
e.g.
```elixir
start(conn, "/")
|> click("foo")
|> tap(fn {_, {_, view, _}} -> open_browser(view) end)
|> assert_html("bar")
```

It's mildly non-ideal to have to juggle the data structure in this case, but it's not a show-stopper and hopefully one off.

Reply:
👋 I'm circling back to say that with more use I completely agree with you. This would be tremendously helpful. I opened #18 and am hoping to find a little time to put more work into this lib 🔜

Reply:
<img src="https://media0.giphy.com/media/3ohzdIuqJoo8QdKlnW/giphy-downsized-medium.gif"/>

Reply:
This has been released in the latest version. Btw, I'm also working on some fairly significant rearrangement to the lib. I'm trying to make it backwards compatible, but just a heads up. Also I could run out of steam before getting it out 😂 See #23

Reply:
@capitalist If you're interested, I've cut an RC for #23 `0.11.0-rc1`. Thought that might help us test-drive the changes.